src/api/system/system.ts

```diff
@@ -606,6 +611,6 @@ export default class System extends Base {
      * @tutorial System.registerExternalConnection
      */
     public registerExternalConnection(uuid: string): Promise<ExternalConnection> {
-        return this.wire.sendAction('register-external-connection', {uuid}).then(({ payload }) => payload.data);
+        return this.wire.sendAction<ExternalConnection>('register-external-connection', {uuid}).then(({ payload }) => payload.data);
```

Comment:
<ExternalConnection> => <ExternalConnection, string>

Reply:
sendAction takes an Identity, not a string.

Reply:
<Identity>